configs/parser_test.go

```diff
@@ -83,13 +83,12 @@ func testNestedModuleConfigFromDir(t *testing.T, path string) (*Config, hcl.Diag
 
 	parser := NewParser(nil)
 	mod, diags := parser.LoadConfigDir(path)
-	assertNoDiagnostics(t, diags)
```

Comment:
This is a little odd: It doesn't really make sense for a helper to call another helper, or to bail out early instead of letting the caller decide what to do with diagnostics. 

There are also only 2 tests using this code, which I eyeballed, and I think I wrote this anyway so it was probably a thoughtless copy-pasta.